frontend/src/components/product/page/ProductPage.tsx

```diff
@@ -16,6 +17,9 @@ const ProductPage: any = () => {
   const [minPrice, setMinPrice] = useState<number>(0);
   const [maxPrice, setMaxPrice] = useState<number>(100000);
 
+  const [currentPage, setCurrentPage] = useState(1);
+  const [productPerPage, setProductPerPage] = useState(2);
```

Comment:
I think `productPerPage` don't need to set in `useState`. We don't change this value later.
it can use default value by itself like 10, 20 per page.